content/en/agent/guide/agent-status-page.md

```diff
@@ -0,0 +1,253 @@
+```text
+    load
+    ----
+      Instance ID: load [OK]
+      Total Runs: 4
+      Metric Samples: Last Run: 6, Total: 24
+      Events: Last Run: 0, Total: 0
+      Service Checks: Last Run: 0, Total: 0
+      Average Execution Time : 6ms
+```
+
+Terms and descriptions:
+
+| Term                   | Description                                                      |
+|------------------------|------------------------------------------------------------------|
+| Instance ID            | The instance ID and the status of the check.                     |
+| Total Runs             | The total number of times the instance has run.                  |
+| Metric Samples         | The number of fetched metrics.                                   |
+| Events                 | The number of triggered events.                                  |
+| Service Checks         | The number of service checks reported.                           |
+| Average Execution Time | The average time taken to run the instance.                      |
+| Last Run               | The number during the last check run.                            |
+| Total                  | The total number since the Agent's most recent start or restart. |
+
+### Config Errors
+
+This section only displays if there are checks with config errors, for example:
+```text
+    test
+    ----
+      Configuration file contains no valid instances
+```
+
+### Loading Errors
+
+This section only displays if there are checks with loading errors, for example:
+```text
+    test
+    ----
+      Core Check Loader:
+        Check test not found in Catalog
+        
+      JMX Check Loader:
+        check is not a jmx check, or unable to determine if it's so
+        
+      Python Check Loader:
+        unable to import module 'test': No module named test
+```
+
+## JMXFetch
+
+This section displays a list of initialized and failed JMX checks even if there are no checks, for example:
+```text
+  Initialized checks
+  ==================
+    no checks
+    
+  Failed checks
+  =============
+    no checks
+```
+
+## Forwarder
+
+The forwarder uses a number of workers to send payloads to Datadog.
+
+If you see the warning `the forwarder dropped transactions, there is probably an issue with your network`, this means that all the workers were busy. You should review your network performance and tune the `forwarder_num_workers` and `forwarder_timeout` options.
+
+### Transactions
+
+This section displays the transactions made by the forwarder, for example:
+```text
+    CheckRunsV1: 2
+    Dropped: 0
+    DroppedOnInput: 0
+    Events: 0
+    HostMetadata: 0
+    IntakeV1: 2
+    Metadata: 0
+    Requeued: 0
+    Retried: 0
+    RetryQueueSize: 0
+    Series: 0
+    ServiceChecks: 0
+    SketchSeries: 0
+    Success: 6
+    TimeseriesV1: 2
+    Errors: 1
+```
+
+Terms and descriptions:
+
+| Term         | Description                                  |
```

Comment:
I don't think the detail of each endpoint if very useful for users (has it can be changed by the configuration). The main info are:
- `Success` number of transaction successfully sent
- `Errors` number of times a transaction failed to be sent and was retried
- `RetryQueueSize` current number of transaction waiting to be retried
- `Retried` number of time we retried transaction
- `DroppedOnInput` number of transaction that were dropped because all worker were busy
- `Dropped` number of transaction dropped because they were retried to many times